go.mod

```diff
@@ -13,23 +13,23 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/mitchellh/mapstructure v1.4.2
-	github.com/onsi/ginkgo/v2 v2.7.0
-	github.com/onsi/gomega v1.24.1
+	github.com/onsi/ginkgo/v2 v2.8.1
+	github.com/onsi/gomega v1.26.0
 	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
 	github.com/openshift/machine-api-operator v0.2.1-0.20220922080915-f7f326323f37
```

Comment:
Are these both up to date?

Reply:
They are not. Do we want to update them? Ran into some issue while doing so.

Reply:
Yeah it would be good to update them if we can, what issues came up?

Reply:
Looked into this and both of these should be up to date.

Reply:
Have you pushed? They both look like they were updated in september?

Reply:
Updated.